src/test/resources/application-test.yaml

```
@@ -25,4 +26,9 @@ management:
       access: unrestricted
   metrics:
     enable:
-      all: true
\ No newline at end of file
+      all: true
+
+naver:
+  api:
+    client-id: dummy-id
+    client-secret: dummy-secret
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)